### ISSUE TITLE
NTRU sample.c requires update

### ISSUE THREAD

**Issue by mouse07410** [open-quantum-safe/liboqs#839]
The following bug is not fixed here. It applies to all of the NTRU `sample.c` files:
```diff
+++ b/crypto_kem/ntruhps2048677/avx2/sample.c
@@ -24,7 +24,7 @@ void PQCLEAN_NTRUHPS2048677_AVX2_sample_fixed_type(poly *r, const unsigned char
         s[4 * i + 0] =                              (u[15 * i + 0] << 2) + (u[15 * i + 1] << 10) + (u[15 * i + 2] << 18) + ((uint32_t) u[15 * i + 3] << 26);
         s[4 * i + 1] = ((u[15 * i + 3] & 0xc0) >> 4) + (u[15 * i + 4] << 4) + (u[15 * i + 5] << 12) + (u[15 * i + 6] << 20) + ((uint32_t) u[15 * i + 7] << 28);
         s[4 * i + 2] = ((u[15 * i + 7] & 0xf0) >> 2) + (u[15 * i + 8] << 6) + (u[15 * i + 9] << 14) + (u[15 * i + 10] << 22) + ((uint32_t) u[15 * i + 11] << 30);
-        s[4 * i + 3] =  (u[15 * i + 11] & 0xfc)       + (u[15 * i + 12] << 8) + (u[15 * i + 13] << 15) + ((uint32_t) u[15 * i + 14] << 24);
+        s[4 * i + 3] =  (u[15 * i + 11] & 0xfc)       + (u[15 * i + 12] << 8) + (u[15 * i + 13] << 16) + ((uint32_t) u[15 * i + 14] << 24);
     }
 
     for (i = 0; i < NTRU_WEIGHT / 2; i++) {
```

See https://github.com/PQClean/PQClean/issues/342
I believe https://github.com/jschanck/ntru.git has it fixed already.

**Comment by mouse07410** [comment]
@dstebila ping?

Also, in addition to the above patch, the computed output values need adjustment (of course).

**Comment by dstebila** [comment]
Have been swamped this week with teaching-related obligations.  Will try to get this by the end of the weekend.

**Comment by dstebila** [comment]
This will be fixed by #835.
